WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -854,46 +813,49 @@ - (void)cancelEditing
     [self.editorView.focusedField blur];
 	
     if ([self.post hasLocalChanges]) {
-        [self showPostHasChangesActionSheet];
+        [self showPostHasChangesAlert];
     } else {
         [self stopEditing];
         [self discardChangesAndUpdateGUI];
     }
 }
 
-- (void)showPostHasChangesActionSheet
+- (void)showPostHasChangesAlert
 {
-	UIActionSheet *actionSheet;
+    UIAlertController *alertController;
+    alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"You have unsaved changes.", @"Title of message with options that shown when there are unsaved changes and the author is trying to move away from the post.")
+                                                          message:nil
+                                                   preferredStyle:UIAlertControllerStyleActionSheet];
+    [alertController addActionWithTitle:NSLocalizedString(@"Keep Editing", @"Button shown if there are unsaved changes and the author is trying to move away from the post.")
+                                  style:UIAlertActionStyleCancel
```

Comment:
It's super cool to see that... `UIAlertController`, on iPad devices, will automatically hide cancel actions